build.gradle

```diff
@@ -340,29 +340,19 @@ allprojects {
   }
 
   plugins.withType(JavaBasePlugin) {
-    File eclipseBuild = project.file('build-eclipse')
-    eclipse.classpath.defaultOutputDir = eclipseBuild
-    if (isEclipse) {
-      // set this so generated dirs will be relative to eclipse build
-      project.buildDir = eclipseBuild
-      // Work around https://docs.gradle.org/current/userguide/java_gradle_plugin.html confusing Eclipse by the metadata
-      // it adds to the  classpath
-      project.file("$buildDir/pluginUnderTestMetadata").mkdirs()
-    }
+    eclipse.classpath.defaultOutputDir = file('build-eclipse')
```

Comment:
Do we still need this? I'm not completely sure what issue this was originally meant to workaround but I think with Buildship it would behoove us to have Gradle and Eclipse share build directories.

Reply:
That would have been my expectation as well, but without it Eclipse still creates the `bin` directories even with build-ship.